test/float/ge_quiet_test.sail

```diff
@@ -0,0 +1,169 @@
+/*==========================================================================*/
+/*     Sail                                                                 */
+/*                                                                          */
+/* Copyright 2024 Intel Corporation                                         */
+/*   Pan Li - pan2.li@intel.com                                             */
+/*                                                                          */
+/* Redistribution and use in source and binary forms, with or without       */
```

Comment:
Unrelated to this PR, but I wonder if we could just use a SPDX identifier instead of all these lines?

Reply:
Sorry, not very sure how to reference to `a SPDX identifier`. Just follow the existing way and put it at the beginning of the source file.
I suppose @Alasdair will replace all in one time for such kind of changes.

Reply:
We probably could, I'm not sure I have the wherewithal to actually go through and change them all now though.

Currently not every file has the same list of copyright holders, and I'm not sure how best we would handle that with just a SPDX identifier?

Reply:
Uploaded https://github.com/rems-project/sail/pull/721